<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Query</class>
 <widget class="QTabWidget" name="Query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Serif</family>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget::pane { /* The tab widget frame */
     border-top: -1px solid #C2C7CB;
 }

 QTabWidget::tab-bar {
     left: 10px; /* move to the right by 10px */

 }

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {

     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border: 2px solid #C4C4C3;
     border-bottom-color: #C2C7CB; /* same as the pane color */
     border-top-left-radius: 4px;
     border-top-right-radius: 15px;
     min-width: 8ex;
     padding: 2px;
	color: rgb(0, 0, 0);

 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,

                                 stop: 0 rgb(31, 90, 175), 
stop: 0.2 rgb(203, 235, 235),
                                 stop: 0.6 rgb(203, 235, 235), stop: 1.0 rgb(31, 90, 175));

 }

 QTabBar::tab:selected {
     border-color: rgb(31, 90, 175);
     /*border-bottom-color: #C2C7CB; same as pane color*/ 
	color: rgb(0, 0, 255);
 }
QLabel{
color:rgb(0, 170, 255); 
}

 QTabBar::tab:!selected {
     margin-top: 2px; /* make non-selected tabs look smaller */
 }

 /* make use of negative margins for overlapping tabs */
 QTabBar::tab:selected {
     /* expand/overlap to the left and right by 4px */
     margin-left: -4px;
     margin-right: -4px;
 }

 QTabBar::tab:first:selected {
     margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
 }

 QTabBar::tab:last:selected {
     margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
 }

 QTabBar::tab:only-one {
     margin: 0; /* if there is only one tab, we don't want overlapping margins */
 }
</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Data Query</string>
   </attribute>
   <widget class="QTableView" name="tableView_TestResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QHeaderView {
    color:rgb(255, 255, 255);
    font: bold 26pt;
    background-color: rgb(85, 170, 255);
    border: 0px solid rgb(144, 144, 144);
    border:0px solid rgb(191,191,191);
    border-left-color: rgba(255, 255, 255, 0);
    border-top-color: rgba(255, 255, 255, 0);
    border-radius:0px;
    min-height:29px;
	 selection-background-color: rgb(77, 77, 77);  
    selection-color: white;  
}
 
QTableView{
color:rgb(0, 85, 255);
}

QScrollBar:horizontal{height:25px;background:transparent;background-color:rgb(249, 249, 249);margin:0px,0px,0px,0px;padding-left:25px;padding-right:25px;}

QScrollBar::handle:horizontal{height:25px;background:lightgray;border-radius:6px;}

QScrollBar::handle:horizontal:hover{height:25px;background:rgb(0, 85, 127);border-radius:6px;}



QScrollBar:vertical{width:25px;background:transparent;background-color:rgb(249, 249, 249);margin:0px,0px,0px,0px;padding-top:25px;padding-bottom:25px;}
QScrollBar::handle:vertical{width:25px;background:lightgray ;border-radius:5px;min-height:20px;}
QScrollBar::handle:vertical:hover{width:25px;background:rgb(0, 85, 127);border-radius:5px;min-height:20px;}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_testDelete_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>630</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Vemana2000</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/image/Clean.png</normaloff>:/image/Clean.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_back">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>665</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_query">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>665</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Query</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_export">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>665</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_delete">
    <property name="geometry">
     <rect>
      <x>383</x>
      <y>665</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Serif</family>
      <pointsize>29</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Sample number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="btn_previous">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Serif</family>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_next">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Serif</family>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_query_SampleNo">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  border: 1px solid rgb(41, 57, 85);
    border-radius: 15px;  
    background: white;  
    selection-background-color: green; 
    font-size: 44px;
 lineedit-password-character: 42;
}
 </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-8</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Time </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QDateEdit" name="dt_query_Start">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>212</width>
       <height>66</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateEdit{

border-width: 1px; 
border-style: solid;  
border-color: rgb(0, 0, 0); 
}

QDateEdit::down-button {
width: 60px;                 

}


QDateEdit::up-button {
width: 60px;                 

}
</string>
     </property>
     <property name="displayFormat">
      <string notr="true">yyyy-MM-dd</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>17</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Serif</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_end">
     <property name="geometry">
      <rect>
       <x>307</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Serif</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dt_query_End">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>211</width>
       <height>66</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateEdit{

border-width: 1px; 
border-style: solid;  
border-color: rgb(0, 0, 0); 
}

QDateEdit::down-button {
width: 60px;                 

}


QDateEdit::up-button {
width: 60px;                 

}
</string>
     </property>
     <property name="displayFormat">
      <string notr="true">yyyy-MM-dd</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_number_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
image: url(:/image/Left.png); 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>598</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
border-radius: 20px;
border: 2px groove gray;
border-style: outset;
}
QPushButton:hover{
background-color:white;
color: black;
}
QPushButton:pressed{
background-color:rgb(0, 0, 0);
color: white;
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/image/arrow_left.png</normaloff>:/image/arrow_left.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_export">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>598</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
border-radius: 20px;
border: 2px groove gray;
border-style: outset;
}
QPushButton:hover{
background-color:white;
color: black;
}
QPushButton:pressed{
background-color:rgb(0, 0, 0);
color: white;
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/image/export.png</normaloff>:/image/export.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_query">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>598</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
border-radius: 20px;
border: 2px groove gray;
border-style: outset;
}
QPushButton:hover{
background-color:white;
color: black;
}
QPushButton:pressed{
background-color:rgb(0, 0, 0);
color: white;
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/image/History.png</normaloff>:/image/History.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>598</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
border-radius: 20px;
border: 2px groove gray;
border-style: outset;
}
QPushButton:hover{
background-color:white;
color: black;
}
QPushButton:pressed{
background-color:rgb(0, 0, 0);
color: white;
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/image/Clean.png</normaloff>:/image/Clean.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
